### ISSUE TITLE
llvm-manpages broken?

### ISSUE THREAD

**Issue by dtzWill** [NixOS/nixpkgs#46854]
## Issue description

On latest master and 18.09 channels I'm seeing this:

```
  unpacking sources
  unpacking source archive /nix/store/krj09whwsicv5hqnzi1qf44r78428h2f-llvm-6.0.1.src.tar.xz
  patching sources
  substituteStream(): ERROR: variable $lib is unset
```

For `llvmPackages_6.llvm-manpages`.

Having never seen this before thought I'd file an issue in case I missed some important change? :)

## Technical details

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the
results.

 - system: `"x86_64-linux"`
 - host os: `Linux 4.18.7, NixOS, 18.09beta302.9fa6a261fb2 (Jellyfish)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.2dtz-mux-6fb06a5`
 - channels(will): `"nur-nopkgs, testing, unstable-19.03pre152504.56b9f6fc8e1, nixos-1609-16.09.1943.25f4906da6"`
 - channels(root): `"nixos-18.09beta302.9fa6a261fb2"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

**Comment by dtzWill** [comment]
Looks like some attributes (postPatch and postInstall) refer to `$lib` output that doesn't exist for the manpage-only variant, I'll submit a PR fixing this in a few.

**Comment by dtzWill** [comment]
Fixed as part of #46888.
